wca-guidelines.md

```diff
@@ -39,6 +39,7 @@ To be more informative, each Guideline is classified using one of the following
 - 1c3++) [ADDITION] Physical records (e.g. score sheets, written solutions from Fewest Moves Solving) must be retained for a minimum of one month. Digital records (e.g. scramble sequences, results) must be retained permanently.
 - 1c3b+) [CLARIFICATION] If there are multiple groups, it is not necessary to identify which competitor was in which groups.
 - 1c4+) [RECOMMENDATION] Corrections to the results should be available within one week of the competition date.
+- 1c8+) [EXAMPLE] The Delegate should only give access to scrambles for future groups to other members of the organization team if they will be temporarily unavailable (e.g. competing themselves), and should give others access to the minimum possible amount of scrambles in this case.
```

Comment:
I think `scrambles` should be `scramble sequences` to have a consistent terminology.

Additionally, why `should` and not `must`?

Reply:
Done. Since this is in the Guidelines, I've also changed this from `[EXAMPLE]` to `[ADDITION]`.